postgres/patches/20240106_remove-cursor-result-id.sql

```diff
@@ -0,0 +1,6 @@
+\connect dstk;
+
+ALTER TABLE dstk_metadata.cursors DROP COLUMN result_id;
```

Comment:
You can just merge this with the original `CREATE TABLE`; this is all pre-version 1 so rules around patch hygiene are more just like friendly suggestions

Reply:
Done!